### PR TITLE
Implemented resistor-color-duo exercise

### PR DESCRIPTION
**Pull request by aes421**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -2,10 +2,6 @@
   "language": "C#",
   "active": true,
   "blurb": "C# is a modern, object-oriented language with lots of great features, such as type-inference and async/await. The tooling is excellent, and there is extensive, well-written documentation.",
-  "online_editor": {
```
Comment (aes421) [inline config.json:5]: @ErikSchierboom The PowerShell script made these changes.  I'm not sure if I should leave it or manually remove it.  Just let me know.

Comment (ErikSchierboom) [reply config.json:5]: Ah yes, the PowerShell script is still a bit outdated. This is a side-effect of us working on Exercism v3. The key should not be removed.

File: exercises/resistor-color-duo/ResistorColorDuo.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class ResistorColorDuo
+{
+    public static int Value(string[] colors) {
+        throw new NotImplementedException("You need to implement this function.");
+    }
```
Comment (ErikSchierboom) [inline exercises/resistor-color-duo/ResistorColorDuo.cs:10]: Tiny formatting suggestion:

```suggestion
    public static int Value(string[] colors)
    {
        throw new NotImplementedException("You need to implement this function.");
    }
```

File: exercises/resistor-color-duo/ResistorColorDuo.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
```
Comment (ErikSchierboom) [inline exercises/resistor-color-duo/ResistorColorDuo.cs:4]: These namespaces don't have to be used by the student, so we can remove them here.

```suggestion
```

File: exercises/resistor-color-duo/Example.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+                                                             .Select(c => Array.IndexOf(AllColors, c))
+                                                      );
+
+    private static int concat(IEnumerable<int> numbers)
+    {
+        return int.Parse(numbers.Aggregate("", (acc, num) => acc + num));
+    }
+
+     private static readonly string[] AllColors = new[] { "black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white" };
+}
```
Comment (ErikSchierboom) [inline exercises/resistor-color-duo/Example.cs:18]: Just to show an alternative way of coding this:

```suggestion
using System;

public static class ResistorColorDuo
{
    public static int Value(string[] colors) => Value(colors[0]) * 10 + Value(colors[1]);

    public static int Value(string color) => Array.IndexOf(AllColors, color);

    private static readonly string[] AllColors = new[] { "black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white" };
}
```

Note that you don't have to update to this version if you prefer yours :)

File: exercises/resistor-color-duo/ResistorColorDuo.csproj [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="Example.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="xunit" Version="2.4.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="coverlet.collector" Version="1.2.0" />
```
Comment (ErikSchierboom) [inline exercises/resistor-color-duo/ResistorColorDuo.csproj:13]: One disadvantage of the `add-exercise.ps1` script is that it doesn't update the packages to the latest version (which we're using in the other exercises). The packages should be:

```suggestion
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
    <PackageReference Include="xunit" Version="2.4.1" />
    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
```

Note that if you feel like fixing this in the `add-exercise.ps1` script, you'd be most welcome!

**Review by ErikSchierboom — APPROVED** [review]
Just a couple of minor comments! Brilliant work.

File: exercises/resistor-color-duo/Example.cs (hunk 4 above)
Comment (aes421) [reply exercises/resistor-color-duo/Example.cs]: I added both examples now!

File: exercises/resistor-color-duo/Example.cs [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+// An Alternative implementation that doesn't utilize LINQ
+// public static class ResistorColorDuo
+// {
+//     public static int Value(string[] colors) => Value(colors[0]) * 10 + Value(colors[1]);
+//     public static int Value(string color) => Array.IndexOf(AllColors, color);
+//     private static readonly string[] AllColors = new[] { "black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white" };
+// }
```
Comment (ErikSchierboom) [inline exercises/resistor-color-duo/Example.cs:26]: I'd just remove this :)

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Thanks @aes421! 🎉

**Pull request merged**